src/items.cpp

```diff
@@ -1224,6 +1226,15 @@ void Items::parseItemNode(const pugi::xml_node& itemNode, uint16_t id)
 							}
 						}
 
+						// datapack compatibility, presume damage to be initialdamage if initialdamage is not declared.
+						// initDamage = 0 (dont override initDamage with damage, dont set any initDamage)
+						// initDamage = -1 (undefined, override initDamage with damage)
+						if (initDamage > 0 || initDamage < -1) {
```

Comment:
< -1? in this case you gonna set opposite, someone gonna write -200 in xml file and you gonna cast it like -initDamage so it will end up in 200, so it will heal not damage.

Reply:
Yes, this is intentional. If you look at my first gif in the above post, with this snippet:
```xml
<attribute key="field" value="fire">
	<attribute key="initdamage" value="-2000" />
	<attribute key="ticks" value="2000" />
	<attribute key="count" value="5" />
	<attribute key="damage" value="400" />
</attribute>
```

Instead of dealing damage, it will heal you 2000 hit points, but then over 2x5 seconds you will loose the obtained 2000 hitpoints. 

Or the opposite:
```xml
<attribute key="field" value="fire">
	<attribute key="initdamage" value="2000" />
	<attribute key="ticks" value="2000" />
	<attribute key="count" value="5" />
	<attribute key="damage" value="-400" />
</attribute>
```
Initdamage will deal 2000 damage, and then it will heal you back over 2x5 seconds. 
This is flexibility. :) You could make custom healing fields.

Reply:
well, you can easily achieve this with simple movement script and condition_regeneration, so I don't know if that's something we should support in condition_damage which says DAMAGE, makes no sense to make it support healing.

Reply:
> well, you can easily achieve this with simple movement script and condition_regeneration, so I don't know if that's something we should support in condition_damage which says DAMAGE, makes no sense to make it support healing.

The damage property already supports this (without this PR). This PR just makes initialdamage consistent with how damage works. (And I think its cool). The opposite of damage is healing, which correlates to a negative damage value. 

If you do not think damage should have the posibility to do healing, it should be considered an issue separately from this PR?

Reply:
well my point is, you can by accident break your server, when you put -200 instead of 200, cause someone might think damage is negative all the time. Imagine having fields that will heal for absurd amounts of hp.